nifi-nar-bundles/nifi-azure-bundle/nifi-azure-reporting-task/src/main/java/org/apache/nifi/reporting/azure/loganalytics/AbstractAzureLogAnalyticsReportingTask.java

```diff
@@ -136,7 +136,7 @@ protected void sendToLogAnalytics(final HttpPost request, final String workspace
 
         final int bodyLength = rawJson.getBytes(UTF8).length;
         final ZonedDateTime zNow = ZonedDateTime.now(ZoneOffset.UTC);
-        final String nowRfc1123 = zNow.format(DateTimeFormatter.RFC_1123_DATE_TIME);
+        final String nowRfc1123 = zNow.format(DateTimeFormatter.ofPattern("EEE, dd MMM yyyy HH:mm:ss O"));
```

Comment:
In taking a closer look, this is correct, and also I noticed a static formatter is defined (and unused) at the top of the class. Can we use that?
```suggestion
        final String nowRfc1123 = zNow.format(RFC_1123_DATE_TIME);
```

Also, perhaps it's worth adding a comment for the static formatter explaining why it is necessary to use that instead of `DateTimeFormatter.RFC_1123_DATE_TIME`, to avoid confusion and possible regression for future maintainers. Something like:

```java
// DateTimeFormatter.RFC_1123_DATE_TIME does not work in every case, such as when a
// two-digit day of month is always required, so we are defining our own formatter here.
private static final DateTimeFormatter RFC_1123_DATE_TIME = DateTimeFormatter
```

Reply:
Funny how the bugfix was there all along, just unused! I hadn't even noticed. I have corrected the code now using your suggestion verbatim :-)